docs/organizations/managing-people.md

```diff
@@ -59,6 +62,9 @@ To add people to your organization:
 
 Members of an organization on Codacy can remove themselves from the organization, and organization admins can also remove other members and committers.
 
+!!! note
+    For Enterprise plans where Codacy automatically manages people activity for your organization, you can only remove inactive people who performed no activity for 90 consecutive days. Inactive people don't occupy a seat in your organization.
+
```

Comment:
I wonder if we could:

* Move this admonition a bit further down as a contextual step to [removing people](https://github.com/codacy/docs/blob/72c7165dea9ae51537b1c3af74b3f5d94cdcaf1f/docs/organizations/managing-people.md?plain=1#L74)
* By the same logic, move the [admonition for adding](https://github.com/codacy/docs/blob/72c7165dea9ae51537b1c3af74b3f5d94cdcaf1f/docs/organizations/managing-people.md?plain=1#L39-L40) Active Pro members to be a contextual step of the [Add People](https://github.com/codacy/docs/blob/72c7165dea9ae51537b1c3af74b3f5d94cdcaf1f/docs/organizations/managing-people.md?plain=1#L44) step, and perhaps even mention the different label (although we'd be breaking our "happy path" rule, but the admonition is there already, so...)

It may just be my preference for moving these right next to the specific step where the user may ask themselves, "why is my experience different?"

Thoughts?

Reply:
Updated! 👍 
I tried different solutions for the "removing people" admonition by moving it around different locations in the flow... Anyway, nothing like another pair of eyes! 👀 😅 
Applied it also for the "adding people".